### ISSUE TITLE
Cinnamon 4.0 changes require your attention

### ISSUE THREAD

**Issue by brownsr** [linuxmint/cinnamon-spices-themes#495]
@germanfr @praveenp @andr35 @cloweling @ubluish @freemangilgamesh  You all have authored themes that are reasonably popular with the Cinnamon user base.  In Cinnamon 4.0 a new grouped window list will be the default, and this has new CSS associated with it.  Your theme(s) will work without changes, but the CSS will fall back to the default Cinnamon theme, which will not be ideal.  As Cinnamon 4.0 is already out on some distros, and will be in the next Mint release which should enter public beta tomorrow, then it would be much appreciated if you could find time to PR some suitable changes.

You might also like to know that as Mint-Y is now going to come with a variety of colour variations then I will be similarly writing to the authors who have adapted the original Mint-Y to colours of their choice, to see if their Mint-Y related theme should be kept.  I'll do that once the Mint Beta is out.

Thanks for all your work in writing themes for Cinnamon users.  

There is no need to reply, but I will be looking out for your PRs with keen interest !

Regards

Simon

**Comment by smurphos** [comment]
@brownsr what are your thoughts on Mint-Y-Yltra. Updating them is on my to do list. 

Reasons for keeping them...so Mint 18.x and 19 users still have a blue Mint-Y and users of other Distros that might not ship a Mint themes package can easily access a Mint-Y without building from source.

**Comment by brownsr** [comment]
@smurphos My working assumption is to remove mint-Y variants, unless they have evolved to be different in some way.  Personally I see no great reason to have a lot of very similar stuff available.  Those on older versions who use them will already have a local copy.  I hadn't thought about distros who might not ship the core mint-themes package.  I guess that's mainly a question for the Cinnamon maintainers of those distros.  I don't know if that is an actual problem or a theoretical one.

**Comment by ubluish** [comment]
@brownsr sorry i won't be able to continue maintaining qob-theme so feel free to change the author to the new maintainer

**Comment by smurphos** [comment]
> @smurphos My working assumption is to remove mint-Y variants in the spices repository

Fine by me - I've got no particular attachment to it, it was just an easy target to update to GTK 3.22 last time round.  Probably time to clean out any remaining GTK3.18 only themes as-well.

**Comment by brownsr** [comment]
@edoz90 You also have a popular theme with Numix Cinnamon Transparent.  See above.

Regards

Simon

**Comment by brownsr** [comment]
@karthikbhupathy13 Your metallic cyan theme is getting some positive feedback.  Are you able to update it for Cinnamon 4.0 ?

Regards

Simon

**Comment by brownsr** [comment]
@Radek71 How about bringing your clean and clear Thunderbolt theme up to date to match Cinnamon 4.0 ?

Regards

Simon

**Comment by edoz90** [comment]
@brownsr: sorry but I do not use Cinnamon anymore

**Comment by brownsr** [comment]
@edoz90  Thanks for letting me know.  I'll mark your theme as not having a current author, if that's OK.

**Comment by edoz90** [comment]
@brownsr yeah sure, no problem!

**Comment by karthikbhupathy13** [comment]
yes workin on it to use it for linux mint 19 cinnamon 3.8.9 version.
do we have cinnamon 4 in use?

Best Regards,
*Karthik Bhupathy*

**Comment by Cloweling** [comment]
Hi, good morning, in fact I already have the theme cinnamon version 4, I have made many changes and I have decided to use the icons and theme that is used in the linux mint official, only in the script to generate the different linux mint themes. I have put the color based on my theme, but I have not yet risen because of the fact that there are still changes that are made.

![screen](https://user-images.githubusercontent.com/22055828/49600603-32805480-f949-11e8-8672-c9c0eadbd220.png)
In the next week I will make the changes, see you later.
